expertsystem/state/particle.py

```diff
@@ -464,8 +662,49 @@ def initialize_graph(  # pylint: disable=too-many-locals
     particles: ParticleCollection,
     initial_state: Sequence[StateDefinition],
     final_state: Sequence[StateDefinition],
-    final_state_groupings: Optional[List[List[List[str]]]] = None,
-) -> List[StateTransitionGraph[dict]]:
+    final_state_groupings: Optional[
+        Union[List[List[List[str]]], List[List[str]], List[str]]
+    ] = None,
+) -> List[StateTransitionGraph[ParticleWithSpin]]:
+    def embed_in_list(some_list: List[Any]) -> List[List[Any]]:
+        if not isinstance(some_list[0], list):
+            return [some_list]
+        return some_list
+
+    allowed_kinematic_groupings = None
+    if final_state_groupings is not None:
+        final_state_groupings = embed_in_list(final_state_groupings)
+        final_state_groupings = embed_in_list(final_state_groupings)
```

Comment:
Does that work if the final state grouping would be of type `List[List[str]]`?

Reply:
Yes, because:
https://github.com/ComPWA/expertsystem/blob/8b0660eda9d2acc817682dab3c1c5a648d48c455/expertsystem/state/particle.py#L670-L671

Reply:
ah yeah ok